### ISSUE TITLE
Figure out CI!

### ISSUE THREAD

**Issue by chmouel** [openshift-pipelines/opc#2]
only e2e tests here, since unit tests is covered atm by the upstream.

we should probably only cover additional e2e tests that are not in upstream/downstream...

we can do kind/github actions but we would not be able to test openshift specific stuff.

It would be nice if we can run them periodic here (on pr) before bumping versions and not just "downstream" before release.

food for thoughts! 

@VeereshAradhya @ppitonak @vdemeester

**Comment by vdemeester** [comment]
Indeed, periodically and possibly on multiple openshift pipelines versions 🙃

**Comment by chmouel** [comment]
We would need clusters budget for that 🙃
